lnrpc/rpc.proto

```diff
@@ -249,12 +249,7 @@ service Lightning {
     /** lncli: `newaddress`
     NewAddress creates a new address under control of the local wallet.
     */
-    rpc NewAddress (NewAddressRequest) returns (NewAddressResponse);
-
-    /**
-    NewWitnessAddress creates a new witness address under control of the local wallet.
-    */
-    rpc NewWitnessAddress (NewWitnessAddressRequest) returns (NewAddressResponse) {
+    rpc NewAddress (NewAddressRequest) returns (NewAddressResponse) {
```

Comment:
The below needs to capture a value such that the user can specify which address type on the REST interface.

Reply:
Tested this as is and the parameter can still be specified, but it's optional. In the case that it isn't specified, the default will be used (P2WPKH).